hibernate-search-elasticsearch/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,284 @@
+        <div class="ui cards">
+            <div class="card" ng-repeat="author in authors">
+                <div class="content">
+                    <div class="header">{{ author.firstName }} {{ author.lastName }}</div>
+                    <div class="meta">
+                        {{ author.books.length }} <i class="book icon"></i>
+                        <button class="ui basic blue icon button mini right floated" ng-click="showBookModal(author)"><i class="plus circle icon"></i> Add book</button>
+                    </div>
+                    <div class="description">
+                        <div ng-repeat="book in author.books">
+                            <div>
+                                {{ book.title }}
+                                <a ng-click="remove('book', book)"><i class="trash alternate outline right floated icon"></i></a>
+                            </div>
+                            <div class="ui divider"></div>
+                        </div>
+                    </div>
+                    <div class="extra content">
+                        <div class="ui two buttons">
+                            <button class="ui basic green button" ng-click="showEditAuthorModal(author)"><i class="pencil alternate icon"></i> Edit</button>
+                            <button class="ui basic red button" ng-click="remove('author', author)"><i class="trash alternate outline icon"></i> Delete</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="card">
+                <div class="content">
+                    <button class="ui blue labeled icon button" ng-click="showAddAuthorModal()"><i class="plus circle icon"></i> Add author</button>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="ui modal" id="modal-author">
+        <i class="close icon"></i>
+        <div class="header">
+            {{ authorModalTitle }}
+        </div>
+        <div class="content">
+            <div class="description">
+                <input type="hidden" name="authorId" ng-model="form.author.id" />
+                <form class="ui form">
+                    <div class="required field">
+                        <label>First name</label>
+                        <input type="text" name="firstName" placeholder="First name" ng-model="form.author.firstName" required>
+                    </div>
+                    <div class="required field">
+                        <label>Last name</label>
+                        <input type="text" name="lastName" placeholder="Last name" ng-model="form.author.lastName" required>
+                    </div>
+                </form>
+            </div>
+        </div>
+        <div class="actions">
+            <button class="ui positive right labeled icon button" ng-click="addOrEditAuthor()">
+                <i class="checkmark icon"></i>
+                Submit
```

Comment:
When adding an author and the list is filtered by a search pattern, the author might not show up. Maybe you could add a notifications that confirms the author was created and reminds it might not appear because of search criteria?

Reply:
I now reset the search.